src/Core/Addon/Module/ModuleManager.php

```diff
@@ -348,8 +352,10 @@ public function disable($name)
         if (!$this->adminModuleProvider->isAllowedAccess(__FUNCTION__, $name)) {
```

Comment:
sounds like this block of code is used multiple times, how about creating a reusable function?

Reply:
This is already a reused function, what do you need more?

Reply:
the `if` block could be one function too: how about `throwIfModuleAccessNotAllowed($name)`?

Reply:
Maybe overkill, more you must pass the __FUNCTION__, $name, translated message, or message, parameters and domain. :/

Reply:
Sounds better to directly send the result of `$this->translator->trans(...)` in that case.

Reply:
Do you really expect another refacto in this PR?
As I'm changing some strings, I'll have to wait for 1.7.5.0 to make them available if we don't accept it for the next minor release.